packages/reverse-exports/tests/reverse-exports.test.ts

```diff
@@ -140,6 +140,24 @@ describe('reverse exports', function () {
       'You tried to reverse exports for the file `./foo.bar` in package `my-addon` but it does not match any of the exports rules defined in package.json. This means it should not be possible to access directly.'
     );
   });
+
+  it('breaks TODO rename this test to something better', function () {
```

Comment:
I know this is my line but I hoped you would replace the todo with a better name once you figured out what the issue was 😂 maybe something like "works with single * replacement" ? 🤔

Reply:
Fixed.